Classes/Service/FileGenerator.php

```diff
@@ -27,6 +27,10 @@
 use TYPO3\CMS\Core\Utility\GeneralUtility;
 use TYPO3\CMS\Fluid\View\StandaloneView;
 
+const OVERWRITE_SETTING_SKIP = 1;
+const OVERWRITE_SETTING_KEEP = 1;
+const OVERWRITE_SETTING_MERGE = 1;
```

Comment:
these constants are unused, right?

Reply:
Yes - first started to add them there and forgot to remove again!